### DISCUSSION TITLE
[Shards] Any hints to start solving the challenge?

### DISCUSSION THREAD

**Discussion by EthanOK** [tinchoabbate/damn-vulnerable-defi#72] [Help / Q&A / Troubleshooting]
I need a little hint.

**Comment by Jayanring** [comment]
Function `cancel` does not behave as expected.

**Comment by tinchoabbate** [comment]
Hi @EthanOK , have you made any progress? Have you thought about what @Jayanring said? Do you want to share here the ideas that you've had? Perhaps that can help others too.
